Dockerfiles/Dockerfile-awsk8s

```diff
@@ -14,28 +17,18 @@ RUN set -eux \
 	&& find /usr/lib/ -name '*.pyc' -print0 | xargs -0 -n1 rm -rf
 
 # kubectl (copied to final image)
+ARG PLATFORM_SHORT
 RUN set -eux \
 	&& curl -sS -L -o /usr/bin/kubectl \
-		https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl \
+		https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/${PLATFORM_SHORT}/kubectl \
 	&& chmod +x /usr/bin/kubectl \
 	&& kubectl version --client --short=true 2>&1 | grep -E 'v[.0-9]+'
 
-# openshift client (copied to final image)
-RUN set -eux \
-	&& curl -sS -L -o /tmp/openshift-client-linux.tar.gz https://mirror.openshift.com/pub/openshift-v4/clients/ocp/stable/openshift-client-linux.tar.gz \
-	&& mkdir /tmp/openshift-client-linux/ \
-	&& tar -xzf /tmp/openshift-client-linux.tar.gz -C /tmp/openshift-client-linux/ \
-	&& mv /tmp/openshift-client-linux/oc /usr/bin/oc-dynamically-linked \
-	&& echo -e '#!/bin/sh\n/lib/ld-musl-x86_64.so.1 --library-path /lib /usr/bin/oc-dynamically-linked "$@"' > /usr/bin/oc \
-	&& chmod +x /usr/bin/oc \
-	&& oc version --client
-
```

Comment:
Need to find source of openshift client for arm64

Reply:
Maybe here? https://mirror.openshift.com/pub/openshift-v3/clients/3.9.8/linux-aarch64/

I've never used openshift ... if anyone can suggest.

Reply:
I've looked around some more and I'm not finding oc arm64 binaries published. I've checked https://github.com/openshift/origin which is used by other openshift-client projects Dockerfile - no arm64. It appears they also push to hub.docker.com/u/openshift - no arm64 for any images 

it does look like rpms for openshift is provided for a large number of archs tho https://www.rpmfind.net/linux/rpm2html/search.php?query=origin-clients

I'm inclined to skip it for arm64 but then would also need to update docs etc